backend/app/api/endpoints/annotations.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""API Endpoints for Media Annotations"""
+
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import APIRouter, Body, Depends, status
+from fastapi.openapi.models import Example
+
+from app.api.dependencies import get_media_id
+from app.api.tags import Tags
+from app.schemas.base import BaseIDModel
+
+router = APIRouter(prefix="/api/media/{media_id}/annotations", tags=[Tags.ANNOTATIONS])
```

Comment:
```suggestion
router = APIRouter(prefix="/api/projects/{project_id}/dataset/items/{dataset_item_id}/annotations", tags=[Tags.ANNOTATIONS])
```
Based on the new design #4624